DependencyInjection/7_PropertyTests.cs

```diff
@@ -19,7 +19,7 @@ namespace FeatureTests.On.DependencyInjection
     [SpecialCase(typeof(SimpleInjectorAdapter), @"
         Simple Injector does not inject properties out of the box, but this behavior 
         can be changed by replacing the `Container.Options.PropertySelectionBehavior`.
-        For more info see: https://bit.ly/1cmMxuS.
+        For more info see: https://bit.ly/14soDfe.

```

Comment:
I suggest using full URLs (for same reasons why StackOverflow suggests it).
If those are too long, then keep the short ones for now and I can look for a solution (e.g. renderer currently supports having a link instead of a tooltip, not sure if I expose it to the test runner at the moment).